modules/glimpse/chunk/meta.yml

```diff
@@ -0,0 +1,84 @@
+name: "glimpse_chunk"
+description: Defines chunks where to run imputation
+keywords:
+  - chunk
+  - imputation
+tools:
+  - "glimpse":
+      description: "GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies."
+      homepage: "https://odelaneau.github.io/GLIMPSE"
+      documentation: "https://odelaneau.github.io/GLIMPSE/commands.html"
+      tool_dev_url: "https://github.com/odelaneau/GLIMPSE"
+      doi: "10.1038/s41588-020-00756-0"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - input:
```

Comment:
The input is called `reference_panel` inside the module, so it's better to also use this here instead of `input`:)

Reply:
Maybe it will be better to do the opposite ?
For people not to mix-up the different argument, I should maybe use the same name as used in the software ?

Reply:
Yeah that sounds like a good idea :)